src/ppom-controller.ts

```diff
@@ -246,12 +248,12 @@ export class PPOMController extends BaseControllerV2<
     blockaidPublicKey,
   }: {
     chainId: string;
-    onNetworkChange: (callback: (networkState: any) => void) => void;
+    onNetworkChange: (callback: (networkState: never) => void) => void;
     messenger: PPOMControllerMessenger;
-    provider: any;
+    provider: Provider;
     storageBackend: StorageBackend;
     securityAlertsEnabled: boolean;
-    onPreferencesChange: (callback: (perferenceState: any) => void) => void;
+    onPreferencesChange: (callback: (preferenceState: never) => void) => void;
```

Comment:
similar question

Reply:
Same as above, the `never` typing enables us to assign any function with a void return type to the callback param. Typing `preferenceState` as anything wider than `never` will *narrow* the range of functions that can be assigned to `callback`.

Reply:
So `never` here gives us the exact same effect as `any`, but without suppressing error messages.

Reply:
Resolving as outdated.